packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -2336,6 +2357,25 @@ class SemanticsConfiguration {
     _hasBeenAnnotated = true;
   }
 
+  /// Whether the semantics node is the root of a subtree for which values
+  /// should be announced.
+  /// 
+  /// See also:
+  ///  * [SemanticsFlag.scopesRoute], for a full description of route scoping.
+  bool get scopesRoute => _hasFlag(SemanticsFlag.scopesRoute);
```

Comment:
Maybe I missed it, but should there be an assert somewhere to ensure that you have explicitChildren set when you set this? Or are there use cases where it is actually ok to not do that?

Reply:
I've add an assert to the semantics render object, since that seems like the right place (can't use the const constructor)